app/_hub/kong-inc/request-termination/index.md

```diff
@@ -69,16 +73,31 @@ params:
       default: "`application/json; charset=utf-8`"
       datatype: string
       description: Content type of the raw response configured with `config.body`.
+    - name: trigger
+      required: false
+      default:
+      description: When not set, the plugin always activates. When set to a string, the plugin will activate exclusively on requests containing either a header or a query parameter named like said string.
+    - name: echo
+      required: false
+      default: false
+      description: When set, the plugin will echo a copy of the request back to the client. The main usecase for this is debugging. It can be combined with `trigger` in order to debug requests on live systems without disturbing real traffic
+
   extra: |
     Once applied, every request (within the configured plugin scope of a Service,
     Route, Consumer, or global) will be immediately terminated by
     sending the configured response.
 
 ---
 
+## New in 2.1.0
+
+- `trigger` config option
+- `echo` config option
```

Comment:
@Tieske - By tomorrow am (EST) would you mind improving the request-termination docs - maybe even reflect the latest changelog entries.

Reply:
If you are referring to the `trigger` and echo config options, they are explained on the same commit, a bit later (the section you are commenting in only "lists the changes" - a changelog of sorts).

Reply:
Got it - thank you. Missed that.